featuregates/client/go.mod

```diff
@@ -0,0 +1,75 @@
+module github.com/vmware-tanzu/tanzu-framework/featuregates/client
+
+go 1.17
+
+replace (
+	github.com/vmware-tanzu/tanzu-framework/apis/cli => ../../apis/cli
+	github.com/vmware-tanzu/tanzu-framework/apis/config => ../../apis/config
+	github.com/vmware-tanzu/tanzu-framework/apis/run => ../../apis/run
+	github.com/vmware-tanzu/tanzu-framework/cli/runtime => ../../cli/runtime
+	sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.1.5
+)
+
+require (
+	github.com/vmware-tanzu/tanzu-framework v0.26.0-dev.0.20220824221239-af5a644ffef7
```

Comment:
This needs a `replace` statement at the top as when running `make modules` I am seeing old version getting downloaded.
```
-- Tidying ./featuregates/controller --
~/Documents/code/tanzu-framework/featuregates/controller ~/Documents/code/tanzu-framework
go: downloading github.com/vmware-tanzu/tanzu-framework v0.26.0-dev.0.20220824221239-af5a644ffef7
~/Documents/code/tanzu-framework
-- Tidying ./featuregates/client --
~/Documents/code/tanzu-framework/featuregates/client ~/Documents/code/tanzu-framework
```